core/assets/bundles/bundle.properties

```diff
@@ -927,6 +927,8 @@ block.inverted-sorter.name = Inverted Sorter
 block.message.name = Message
 block.illuminator.name = Illuminator
 block.illuminator.description = A small, compact, configurable light source. Requires power to function.
+block.large-illuminator.name = Large illuminator
+block.large-illuminator.description = A larger version of the illuminator. Provides configurable light for a larger area. Requires power to function.
```

Comment:
For the French translation: `block.large-illuminator.description = Un plus grande version de l'illuminateur. Il fournit une lumière configurable sur une surface large. Il requiert de l'énergie pour fonctionner.`